javagen/src/main/java/com/azure/autorest/util/ModelExampleUtil.java

```diff
@@ -179,7 +179,11 @@ public static ExampleNode parseNode(IType type, IType wireType, Object objectVal
                 node = defaultNode(type, wireType, objectValue);
             }
         } else if (objectValue == null) {
-            node = null;
+            if (ClassType.CONTEXT.equals(type)) {
+                node = new LiteralNode(ClassType.CONTEXT, "").setLiteralsValue("");
```

Comment:
Why we need this? I think we already had special handling for `ClassType.CONTEXT`.

Reply:
Got it. It should already be handled in `parseNodeFromParameter` before entering `parseNode`.